### PR TITLE
Enable Activity tracking on generic host

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

**Comment by tarekgh** [comment]
@shirhatti could you please add a little test for the generic host with logging the activity Ids?

**Comment by tarekgh** [comment]
CC @davidfowl

**Comment by tarekgh** [comment]
CC @maryamariyan

**Review by tarekgh — APPROVED** [review]
Thanks for getting this done!
I added a little request for adding a test. otherwise, LGTM.

**Comment by tarekgh** [comment]
https://github.com/dotnet/runtime/issues/34305
https://github.com/dotnet/runtime/pull/37092

**Review by maryamariyan — APPROVED** [review]

**Review by davidfowl — APPROVED** [review]

File: src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs [hunk 1]
```diff
@@ -60,6 +61,37 @@ public void CreateDefaultBuilder_RegistersEventSourceLogger()
                 args.Payload.OfType<string>().Any(p => p.Contains("Request starting")));
         }
 
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/34580", TestPlatforms.Windows, TargetFrameworkMonikers.Netcoreapp, TestRuntimes.Mono)]
```
Comment (shirhatti) [inline src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs:65]: Not sure if this is still needed, but all Hosting tests had this attribute

**Comment by maryamariyan** [comment]
@saurabh500 some test failures showing up on your last commit: https://dev.azure.com/dnceng/public/_build/results?buildId=698089&view=ms.vss-test-web.build-test-results-tab&runId=21632144&resultId=101794&paneView=debug

File: src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs (hunk 1 above)
Comment (maryamariyan) [reply src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs:65]: That is to skip running tests on mono. They should be tracked with https://github.com/dotnet/runtime/issues/34582 now that 34580 is closed as its dupe.

Comment (tarekgh) [reply src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/HostTests.cs:65]: if the test still fail on Mono, maybe can use issue #34582 instead.

---
In reply to: [443738630](https://github.com/dotnet/runtime/pull/37892#discussion_r443738630) [](ancestors = 443738630)

**Review by tarekgh — APPROVED** [review]
LGTM. please let me know if you still seeing any related CI failures.

**Comment by tarekgh** [comment]
@Anipik does this change will be included in previews 7?  I am asking because I don't know if you already snapped for it yet.

**Comment by Anipik** [comment]
Yep it made to preview7

**Pull request merged**